libraries/AP_ICEngine/AP_ICEngine.h

```diff
@@ -122,12 +126,23 @@ class AP_ICEngine {
 
     enum class Options : uint16_t {
         DISABLE_IGNITION_RC_FAILSAFE=(1U<<0),
+        DISABLE_REDLINE_GOVERNOR = (1U << 1),
     };
     AP_Int16 options;
 
     // start_chan debounce
     uint16_t start_chan_last_value = 1500;
     uint32_t start_chan_last_ms;
+
+    // redline rpm
+    AP_Int32 redline_rpm;
+    struct RedlineValues {
```

Comment:
There should only be one of these so might as well just define and declare it without a name.

```suggestion
    struct RedlineValues {
        bool flag;
        float governor_integrator;
        float throttle_percentage;
    };

    RedlineValues redline;
````
    struct {
        bool flag;
        float governor_integrator;
        float throttle_percentage;
    } redline;

Reply:
Fixed.